apps/api/src/server/repositories/representation.repository.js

```diff
@@ -53,6 +53,86 @@ export const getByCaseId = async (caseId, { page, pageSize }, { searchTerm, filt
 	};
 };
 
+/**
+ *
+ * @param {number} id
+ * @param {number?} caseId
+ * @returns {Promise<any>}
+ */
+export const getById = async (id, caseId) => {
+	let caseFilter = {};
+
+	if (caseId) {
+		caseFilter = {
+			case: {
+				id: caseId
+			}
+		};
+	}
+
+	const representations = await databaseConnector.representation.findMany({
```

Comment:
Just an FYI, if you are looking to find only the first record in the DB for a certain table, then you can use the `findFirst` function instead of `findMany()[0]`